doc/source/array.rst

```diff
@@ -80,6 +80,12 @@ The :class:`GPUArray` Array Class
 
```

Comment:
Commenting here since Github won't let me do so in the correct place: It seems like this PR contains a commit change for the "compyte" submodule. If that's intentional, could you explain? If it's accidental, could you revert it?

Reply:
Thanks for pointing this out - it's an accident resulting from something I don't understand about submodules - I originally authored the commit based on a version of master prior to a compyte update, then after rebasing this morning I somehow ended up with this change as part of my commit. Will revert this change.

Reply:
I think this accidental change is gone now.